runtimes/common/src/constants.rs

```diff
@@ -491,6 +492,7 @@ pub mod tips {
 		pub const TipCountdown: BlockNumber = DAYS;
 		pub const TipFindersFee: Percent = Percent::from_percent(0);
 		pub const TipReportDepositBase: Balance = deposit(1, 1);
+		pub const MaxTipAmount: Balance = 1_000_000 * KILT;
```

Comment:
No, I just didn't want to restrict us, since this is just the max possible amount. I updated the value to 100000: [95661b8](https://github.com/KILTprotocol/kilt-node/pull/655/commits/95661b876f45169fef459a5a1f7e67d6e5d45f2c)

Reply:
Yes but still, the tip committee being able to hand out 1 million KILTs feels like a lot to me. Let's have it lower, and perhaps change it later on.